docs/release-notes/release-notes-0.17.0.md

```diff
@@ -154,6 +154,9 @@ unlock or create.
 
 * Fixed a memory leak found in mempool management handled by
   [`btcwallet`](https://github.com/lightningnetwork/lnd/pull/7767).
+  
+* Make sure to [set the dont_forward msg flag in the channel update msg 
+  for unannounced channels](https://github.com/lightningnetwork/lnd/pull/7759).
```

Comment:
Question: Is this change just addressing unannounced channels or zero conf channels as well?

Reply:
Yes also public zero-conf channels which should be unannounced until they reach a 6 conf target. 

So when we talk about unannounced channels they are either:

unannounced channels alias "private channels" where when initiating the funding process the initiator does not set the `announce_channel ` bit  in the (open_channel msg)[https://github.com/lightning/bolts/blob/master/02-peer-protocol.md#channel-establishment] named [byte:channel_flags].

or zero-conf channel where the previous described bit is set (signaling a announced channel alias "public channel") but the channel has still not reached the 6 confirmation target.

Both of these cases handle unannounced channel therefore being part of the PR.